### ISSUE TITLE
'self' used in method call 'onBoth' before 'super.init' call

### ISSUE THREAD

**Issue by SameeHsn** [barnaclejive/FaceTrigger#3]
I just tried to compile the project but there seem to be some minor issues and I don't know the cause. Help? 

Please see the screenshot below:
<img width="841" alt="screenshot 2018-08-15 at 2 13 48 pm" src="https://user-images.githubusercontent.com/30621868/44141077-7f32a574-a095-11e8-98ab-2781cbb67681.png">

File: FaceTriggeresEvaluators.swift

Thanks!

**Comment by rhenz** [comment]
were you able to resolve this @starptr ?

**Comment by shikhak** [comment]
I am facing the same issue. Tried different ways. couldn't resolve it. Any idea?

**Comment by SameeHsn** [comment]
No. Actually didn't try much to resolve it. Please let me know if you get it.

**Comment by barnaclejive** [comment]
Thanks for the reports all - I'm going take a look and update the project for iOS 12 / Swift 4.2.

**Comment by aloveric** [comment]
same issue
